### PR TITLE
systemtools: addon update (117)

### PR DESCRIPTION
**Pull request by heitbaum**
Update to version 117
- libmtp: update to 1.1.18
  - from 1.1.13
- vim: update to 8.2.2293
  - from 8.2.0023
- usb-modeswitch: update to 2.6.1
  - from 2.5.2
- stress-ng: update to 0.12.01
  - from 0.11.14
- strace: update to 5.10
  - from 5.9
- smartmontools: update to 7.2
  - from 7.0
- inotify-tools: update to 3.20.11.0
  - from 3.20.1
- i2c-tools: update to 3.1.2+patches
  - from 3.1.2, 2 patches, fixed site and URL
- htop: update to 3.0.4
  - from 3.0.0beta5
- file: cleanup version
  - use version not githash - it is the same version.
- fdupes: update to 2.1.2
  - from 2.0.0
- oniguruma: update to 6.9.6
  - from 6.9.1
- libssh2: update to 1.9.0
  - from 1.8.0

### CODE REVIEW

**Comment by heitbaum** [comment]
@CvH - I have included your commit for file in this virtual system-tools PR (hope that’s ok.)

File: packages/addons/addon-depends/system-tools-depends/i2c-tools/package.mk [hunk 1]
```diff
@@ -3,10 +3,10 @@
 
 PKG_NAME="i2c-tools"
 PKG_VERSION="3.1.2"
-PKG_SHA256="f939a6242c03950cc568d0efdfa3db7a9c29e0e8f5abd01f2908bdd344c054ff"
+PKG_SHA256="6d6079153cd49a62d4addacef4c092db1a46ba60b2807070a3fbe050262aef87"
```
Comment (CvH) [inline packages/addons/addon-depends/system-tools-depends/i2c-tools/package.mk:6]: there is a much more recent version, likely they include already the patches
https://mirrors.edge.kernel.org/pub/software/utils/i2c-tools/i2c-tools-4.2.tar.xz

Comment (heitbaum) [reply packages/addons/addon-depends/system-tools-depends/i2c-tools/package.mk:6]: I had planned to bump lm_sensors and i2c-tools together.

https://i2c.wiki.kernel.org/index.php/I2C_Tools

```
i2c-tools                           | 3.1.2                                    | 4.2
lm_sensors                          | Commit date: 2018-11-23                  | Master: 2020-12-0
```

File: packages/addons/tools/system-tools/changelog.txt [hunk 2]
```diff
@@ -1,3 +1,18 @@
...
+- stress-ng: update to 0.12.01
+- strace: update to 5.10
+- smartmontools: update to 7.2
+- inotify-tools: update to 3.20.11.0
+- i2c-tools: update to 3.1.2+patches
+- htop: update to 3.0.4
+- file: cleanup version
+- fdupes: update to 2.1.2
+- oniguruma: update to 6.9.6
+- libssh2: update to 1.9.0
```
Comment (CvH) [inline packages/addons/tools/system-tools/changelog.txt:14]:
```suggestion
117
- fdupes: update to 2.1.2
- file: cleanup version
- htop: update to 3.0.4
- i2c-tools: update to 3.1.2+patches
- inotify-tools: update to 3.20.11.0
- libmtp: update to 1.1.18
- libssh2: update to 1.9.0
- oniguruma: update to 6.9.6
- smartmontools: update to 7.2
- strace: update to 5.10
- stress-ng: update to 0.12.01
- usb-modeswitch: update to 2.6.1
- vim: update to 8.2.2293
```

File: packages/addons/tools/system-tools/package.mk [hunk 3]
```diff
@@ -110,7 +110,7 @@ addon() {
 
     # jq
     cp -P $(get_install_dir jq)/usr/bin/jq $ADDON_BUILD/$PKG_ADDON_ID/bin
-    cp -P $(get_install_dir oniguruma)/usr/lib/libonig.so $ADDON_BUILD/$PKG_ADDON_ID/lib
+    cp -P $(get_install_dir oniguruma)/usr/lib/libonig.so* $ADDON_BUILD/$PKG_ADDON_ID/lib
```
Comment (CvH) [inline packages/addons/tools/system-tools/package.mk:113]: better just copy what is really necessary + avoid copy new stuff in future that nobody needs
example
`cp -P $(get_install_dir oniguruma)/usr/lib/{libonig.so,libonig.so.2,libonig.so.12}`

**Review by CvH — CHANGES_REQUESTED** [review]

File: packages/addons/tools/system-tools/changelog.txt (hunk 2 above)
Comment (CvH) [reply packages/addons/tools/system-tools/changelog.txt:14]: btw just name stuff that anyone might have an interest in, so cleanup version etc is nothing that anyone cares :)

File: packages/addons/tools/system-tools/package.mk (hunk 3 above)
Comment (heitbaum) [reply packages/addons/tools/system-tools/package.mk]: Got it - done.

**Review by CvH — APPROVED** [review]

**Pull request merged**